Assets/MixedRealityToolkit/Inspectors/Profiles/MixedRealitySpatialAwarenessSystemProfileInspector.cs

```diff
@@ -114,60 +100,55 @@ private void RenderList(SerializedProperty list)
                     SerializedProperty observerProfile = observer.FindPropertyRelative("observerProfile");
                     SerializedProperty runtimePlatform = observer.FindPropertyRelative("runtimePlatform");
 
-                    using (new EditorGUILayout.VerticalScope())
+                    using (new EditorGUILayout.VerticalScope(EditorStyles.helpBox))
                     {
                         using (new EditorGUILayout.HorizontalScope())
                         {
                             observerFoldouts[i] = EditorGUILayout.Foldout(observerFoldouts[i], observerName.stringValue, true);
-
+ 
                             if (GUILayout.Button(RemoveObserverContent, EditorStyles.miniButtonRight, GUILayout.Width(24f)))
                             {
                                 list.DeleteArrayElementAtIndex(i);
                                 serializedObject.ApplyModifiedProperties();
                                 changed = true;
                                 break;
                             }
-
                         }
 
-                        if (observerFoldouts[i] || RenderAsSubProfile)
+                        // TODO: TROY
+                        //if (observerFoldouts[i]) || RenderAsSubProfile)
```

Comment:
Same comment on this TODO as well.

Reply:
This comment is on an outdated version of the file. It has already been removed